gridsome/lib/app/loadConfig.js

```diff
@@ -80,8 +80,15 @@ module.exports = (context, options = {}) => {
   config._pathPrefix = normalizePathPrefix(localConfig.pathPrefix)
   config.publicPath = config.pathPrefix ? `${config.pathPrefix}/` : '/'
   config.staticDir = resolve('static')
-  config.outDir = resolve(localConfig.outDir || 'dist')
-  config.assetsDir = path.join(config.outDir, assetsDir)
+  
+   // TODO: remove outDir before 1.0
+  config.outputDir = resolve(localConfig.outputDir || localConfig.outDir || 'dist')
+  if (localConfig.outDir) {
+    config.outDir = config.outputDir
+    deprecate.property(config, 'outDir', 'The outDir config is renamed to outputDir.')
```

Comment:
I think you can add this without the if condition because plugins use it regardless of being set in `gridsome.config.js`. But everything else looks good to me :)

Reply:
Should be done now :)